test/compose.yaml

```
@@ -0,0 +1,14 @@
+services:
+  mongo1:
+    image: 'mongo:5'
+    container_name: mongo1
```

Nitpick - we have only 1 mongo instance, maybe just call it `mongo`? 😉 
```suggestion
  mongo:
    image: 'mongo:5'
    container_name: mongo
```